Integrations tests - Semantics Tests failure

Run on Windows 10:
```
[INFO] --- bnd-testing-maven-plugin:4.2.0:testing (default) @ org.openhab.core.semantics.tests ---
[INFO] Matching glob *
20:24:54.175 [main] DEBUG org.eclipse.smarthome.core.common.ThreadPoolManager - Created thread pool 'safeCall' with size 5
20:24:54.238 [main] DEBUG org.eclipse.smarthome.core.internal.i18n.I18nProviderImpl - No language set, setting locale to 'null'.
20:24:54.238 [main] DEBUG org.eclipse.smarthome.core.internal.i18n.I18nProviderImpl - Error setting measurement system for value 'null'.
20:24:54.300 [main] DEBUG org.eclipse.smarthome.core.common.ThreadPoolManager - Created scheduled thread pool 'scheduler' of size 5
20:24:54.332 [main] DEBUG org.eclipse.smarthome.core.common.ThreadPoolManager - Created thread pool 'ConfigStatusService' with size 5
20:24:54.332 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.eclipse.smarthome.core.semantics.internal.SemanticsMetadataProvider" has been added.
20:24:54.363 [main] DEBUG org.eclipse.smarthome.storage.json.internal.JsonStorageService - Json Storage Service: Activated.
20:24:54.363 [main] DEBUG org.eclipse.smarthome.storage.json.internal.JsonStorage - Json storage file at 'D:\dev\openhab2\git\openhab-core\itests\org.openhab.core.semantics.tests\target\test\tmp\testing\itest\userdata\jsondb\org.eclipse.smarthome.core.items.Metadata.json' does not exist - checking for a backup.
20:24:54.378 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.eclipse.smarthome.core.internal.items.ManagedMetadataProviderImpl" has been added.
20:24:54.378 [main] DEBUG org.eclipse.smarthome.storage.json.internal.JsonStorage - Json storage file at 'D:\dev\openhab2\git\openhab-core\itests\org.openhab.core.semantics.tests\target\test\tmp\testing\itest\userdata\jsondb\org.eclipse.smarthome.core.items.Item.json' does not exist - checking for a backup.
20:24:54.378 [main] DEBUG org.eclipse.smarthome.core.common.registry.AbstractRegistry - Provider "org.eclipse.smarthome.core.items.ManagedItemProvider" has been added.
TEST testByLabelOrSynonym(org.eclipse.smarthome.core.semantics.SemanticTagsTest) <<< ERROR: null
java.util.NoSuchElementException
        at java.util.ArrayList$Itr.next(ArrayList.java:854)
        at org.eclipse.smarthome.core.semantics.SemanticTagsTest.testByLabelOrSynonym(SemanticTagsTest.java:75)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:38)
        at junit.framework.TestSuite.runTest(TestSuite.java:252)
        at junit.framework.TestSuite.run(TestSuite.java:247)
        at aQute.junit.Activator.test(Activator.java:340)
        at aQute.junit.Activator.automatic(Activator.java:236)
        at aQute.junit.Activator.run(Activator.java:177)
        at aQute.launcher.Launcher.lambda$serviceChanged$0(Launcher.java:1385)
        at aQute.launcher.Launcher.run(Launcher.java:352)
        at aQute.launcher.Launcher.main(Launcher.java:152)
TEST testByLabel(org.eclipse.smarthome.core.semantics.SemanticTagsTest) <<< ERROR: expected:<interface org.eclipse.smarthome.core.semantics.model.location.Kitchen> but was:<null>
java.lang.AssertionError: expected:<interface org.eclipse.smarthome.core.semantics.model.location.Kitchen> but was:<null>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:118)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at org.eclipse.smarthome.core.semantics.SemanticTagsTest.testByLabel(SemanticTagsTest.java:68)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:38)
        at junit.framework.TestSuite.runTest(TestSuite.java:252)
        at junit.framework.TestSuite.run(TestSuite.java:247)
        at aQute.junit.Activator.test(Activator.java:340)
        at aQute.junit.Activator.automatic(Activator.java:236)
        at aQute.junit.Activator.run(Activator.java:177)
        at aQute.launcher.Launcher.lambda$serviceChanged$0(Launcher.java:1385)
        at aQute.launcher.Launcher.run(Launcher.java:352)
        at aQute.launcher.Launcher.main(Launcher.java:152)
TEST testGetItemsInLocation(org.eclipse.smarthome.core.semantics.internal.SemanticsServiceImplTest) <<< ERROR: null
java.lang.AssertionError
        at org.junit.Assert.fail(Assert.java:86)
        at org.junit.Assert.assertTrue(Assert.java:41)
        at org.junit.Assert.assertTrue(Assert.java:52)
        at org.eclipse.smarthome.core.semantics.internal.SemanticsServiceImplTest.testGetItemsInLocation(SemanticsServiceImplTest.java:89)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:38)
        at junit.framework.TestSuite.runTest(TestSuite.java:252)
        at junit.framework.TestSuite.run(TestSuite.java:247)
        at aQute.junit.Activator.test(Activator.java:340)
        at aQute.junit.Activator.automatic(Activator.java:236)
        at aQute.junit.Activator.run(Activator.java:177)
        at aQute.launcher.Launcher.lambda$serviceChanged$0(Launcher.java:1385)
        at aQute.launcher.Launcher.run(Launcher.java:352)
        at aQute.launcher.Launcher.main(Launcher.java:152)
Tests run  : 7
Passed     : 4
Errors     : 3
Failures   : 0
[INFO] 3 Error(s)
```

--- a/bundles/org.openhab.core.semantics/src/main/java/org/eclipse/smarthome/core/semantics/SemanticTags.java
+++ b/bundles/org.openhab.core.semantics/src/main/java/org/eclipse/smarthome/core/semantics/SemanticTags.java
@@ -19,6 +19,7 @@ import java.util.Locale;
 import java.util.Map;
 import java.util.MissingResourceException;
 import java.util.ResourceBundle;
+import java.util.ResourceBundle.Control;
 import java.util.Set;
 import java.util.TreeMap;
 import java.util.stream.Collectors;
@@ -31,7 +32,6 @@ import org.eclipse.smarthome.core.semantics.model.Tag;
 import org.eclipse.smarthome.core.semantics.model.TagInfo;
 import org.eclipse.smarthome.core.semantics.model.equipment.Equipments;
 import org.eclipse.smarthome.core.semantics.model.location.Locations;
-import org.eclipse.smarthome.core.semantics.model.point.Control;
 import org.eclipse.smarthome.core.semantics.model.point.Measurement;
 import org.eclipse.smarthome.core.semantics.model.point.Points;
 import org.eclipse.smarthome.core.semantics.model.property.Properties;
@@ -81,7 +81,8 @@ public class SemanticTags {
     }
 
     public static List<String> getLabelAndSynonyms(Class<? extends Tag> tag, Locale locale) {
-        ResourceBundle rb = ResourceBundle.getBundle(TAGS_BUNDLE_NAME, locale);
+        ResourceBundle rb = ResourceBundle.getBundle(TAGS_BUNDLE_NAME, locale,
+                Control.getNoFallbackControl(Control.FORMAT_PROPERTIES));
         try {
             String entry = rb.getString(tag.getAnnotation(TagInfo.class).id());
             return Arrays.asList(entry.toLowerCase(locale).split(","));
@@ -91,7 +92,8 @@ public class SemanticTags {
     }
 
     public static String getLabel(Class<? extends Tag> tag, Locale locale) {
-        ResourceBundle rb = ResourceBundle.getBundle(TAGS_BUNDLE_NAME, locale);
+        ResourceBundle rb = ResourceBundle.getBundle(TAGS_BUNDLE_NAME, locale,
+                Control.getNoFallbackControl(Control.FORMAT_PROPERTIES));
         try {
             String entry = rb.getString(tag.getAnnotation(TagInfo.class).id());
             if (entry.contains(",")) {
@@ -124,7 +126,7 @@ public class SemanticTags {
             if (stateDescription != null && stateDescription.isReadOnly()) {
                 return Measurement.class;
             } else {
-                return Control.class;
+                return org.eclipse.smarthome.core.semantics.model.point.Control.class;
             }
         } else {
             return null;

